PsfLauncher/main.cpp

```diff
@@ -50,10 +44,7 @@ void LogString(const char name[], const wchar_t value[]) noexcept;
 void LogString(const char name[], const char value[]) noexcept;
 void Log(const char fmt[], ...) noexcept;
 
-void RunScript(const psf::json_object& scriptInformation, std::filesystem::path packageRoot, LPCWSTR dirStr, int cmdShow);
-void StartProcess(LPCWSTR applicationName, LPWSTR commandLine, LPCWSTR currentDirectory, int cmdShow, bool runInVirtualEnvironment);
 void StartWithShellExecute(std::filesystem::path packageRoot, std::filesystem::path exeName, std::wstring exeArgString, LPCWSTR dirStr, int cmdShow);

```

Comment:
>StartWithShellExecute [](start = 5, length = 21)

Since you already have a startprocess header, maybe you can move this in to that.